node/src/crypto/hash.rs

```diff
@@ -183,15 +168,15 @@ mod test {
         let inputs_and_digests = [
             (
                 "",
-                "0e5751c026e543b2e8ab2eb06099daa1d1e5df47778f7787faab45cdf12fe3a8",
+                "0E5751C026E543B2E8aB2eb06099dAA1d1e5dF47778F7787fAAB45Cdf12fE3A8",
```

Comment:
These test vectors should work identically for Blake2bHash and Digest.  We probably should just get rid of one of the since they are the same thing…

Reply:
marking as resolved since there is another PR that will unify `Blake2b` and `Digest`.